<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HMagazzino</class>
 <widget class="QWidget" name="HMagazzino">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magazzino</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Resources/Links.png</normaloff>:/Resources/Links.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QDateEdit" name="dtDateFrom"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Al:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDateEdit" name="dtDateTo"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dal:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Filtra</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Search.PNG</normaloff>:/Search.PNG</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btsearch">
         <property name="text">
          <string>Cerca</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Resources/Search.PNG</normaloff>:/Resources/Search.PNG</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>Operazione</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rblot">
         <property name="text">
          <string>Lotto</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbprod">
         <property name="text">
          <string>Prodotto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leLot"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Nuova operazione</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Resources/Plus.PNG</normaloff>:/Resources/Plus.PNG</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>&amp;Annulla Modifiche</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Resources/Banned-Sign.PNG</normaloff>:/Resources/Banned-Sign.PNG</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Salva</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Resources/save-icon.png</normaloff>:/Resources/save-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>&amp;Chiudi</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/Resources/Actions-window-close-icon.png</normaloff>:/Resources/Actions-window-close-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionNew_operation">
   <property name="text">
    <string>new operation</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>HMagazzino</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>751</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>757</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLot</sender>
   <signal>textChanged(QString)</signal>
   <receiver>HMagazzino</receiver>
   <slot>cercaLot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rblot</sender>
   <signal>clicked()</signal>
   <receiver>HMagazzino</receiver>
   <slot>cercaLot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>702</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>697</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbprod</sender>
   <signal>clicked()</signal>
   <receiver>HMagazzino</receiver>
   <slot>cercaLot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_operation</sender>
   <signal>triggered()</signal>
   <receiver>HMagazzino</receiver>
   <slot>cercaLot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cercaLot()</slot>
 </slots>
</ui>
